crates/ra_syntax/src/ast/generated.rs

```diff
@@ -108,6 +108,10 @@ impl ArrayExpr {
     pub fn exprs(&self) -> impl Iterator<Item = &Expr> {
         super::children(self)
     }
+
+    pub fn repeat(&self) -> Option<&Expr> {
+        super::child_opt(self)
```

Comment:
This is a problem: this'll pick the first child of type `Expr`, so it will be the intializer, and not the repeat. I think we have to hand-write these implementations in ast/expr_extensions.rs, like this:

```
enum ArrayExprKind {
    Repeat {Initializer: Option<&Expr>, Repeat: Option<&Expr> },
    ElementList(AstChildren<Expr>),
}

impl ArrayExpr {
    pub fn kind() -> ArrayExprKind;
    fn is_repeat() -> bool {} // look for `;` token, using children_with_tokens. 
    fn exprs() // just all expressions, will use it to implement kind, by looking and nth(2).
}
```